tmpl/package.tmpl

```diff
@@ -24,9 +24,9 @@ package {{.Name}}
 {{define "paramsGoDecl"}}{{range $i, $p := .}}{{if ne $i 0}}, {{end}}{{$p.GoName}} {{$p.Type.GoType}}{{end}}{{end}}
 {{define "paramsGoCall"}}{{range $i, $p := .}}{{if ne $i 0}}, {{end}}{{$p.Type.ConvertGoToC $p.GoName}}{{end}}{{end}}
 
-// #cgo darwin  LDFLAGS: -framework OpenGL
-// #cgo linux   LDFLAGS: -lGL
-// #cgo windows LDFLAGS: -lopengl32
+// #cgo darwin        LDFLAGS: -framework OpenGL
+// #cgo freebsd linux LDFLAGS: -lGL
```

Comment:
Can you reorder them so linux comes before freebsd? Linux is more popular and well supported.